src/OpenLoco/Graphics/PaletteMap.cpp

```diff
@@ -103,23 +73,24 @@ namespace OpenLoco::Gfx
 
         if (image.hasSecondary())
         {
-            // A secondary paletteMap is made up by combinging bits from two palettes.
-            PaletteMap customMap = PaletteMap::getDefault();
+            // Combines portions of two different palettes into the global palette map.
+            auto& paletteMap = getPaletteMapBuffer();
```

Comment:
Isn't this modifying the default? Previously we copied here as we wanted to leave default unchanged.

Reply:
The PaletteMap does not store the data so that copy was not doing what you think it did.